rosbag2_transport/test/rosbag2_transport/test_record_all.cpp

```diff
@@ -150,6 +163,14 @@ TEST_F(RecordIntegrationTestFixture, published_messages_from_topic_and_service_a
 
   ASSERT_TRUE(client_manager_1->wait_for_service_to_be_ready());
 
+  // At this point, we expect that the service /test_service_1, along with the topic /string_topic,
+  // along with the event topic /test_service_1, along with the split topic /events/write_split are
+  // available to be recorded.  However, wait_for_matched() and wait_for_service_to_be_ready() only
+  // check on the service and the topic, not the event or the split topic, so ask the recorder to
+  // make sure it has successfully subscribed to all.
+  ASSERT_TRUE(recorder->wait_for_topic_to_be_discovered("/test_service/_service_event"));
+  ASSERT_TRUE(recorder->wait_for_topic_to_be_discovered("/events/write_split"));
```

Comment:
IMO wait for `"/events/write_split"` is redundant here. We are not going to use it in this test.

Reply:
Removed in 83b0b6b